README.markdown

```diff
@@ -1,23 +1,37 @@
-DistrictBuilder
-===============
+# DistrictBuilder
 
 [![Build Status](https://travis-ci.org/PublicMapping/DistrictBuilder.svg?branch=develop)](https://travis-ci.org/PublicMapping/DistrictBuilder)
 
 [DistrictBuilder](http://www.districtbuilder.org/) is software created by the [Public Mapping Project](http://www.publicmapping.org/resources/software).
 
 The development environment is docker-compose for services inside a Vagrant virtual machine.
 
+## Table of Contents
+  * [Installation](#installation)
+    * [Requirements](#installation-requirements)
+    * [Development Environment Setup](#development-environment-setup)
+      * [`config.xml`](#configxml)
+      * [Setting Up Your Application](#setting-up-your-application)
+  * [Translations](#translations)
+  * [Support](#support)
+  * [Development](#development)
+  * [Deployment](#deployment)
+    * [Requirements](#deployment-requirements)
+    * [Setup](#setup)
+    * [Setting Up the Production Environment](#setting-up-the-production-environment)
 
-Installation
-------------
 
-### Requirements ###
+# Installation
+<h3 id=installation-requirements>
```

Comment:
Fixed. I initially used the HTML syntax because `#requirements` was clashing between the Development and Production `Requirements` sections. Turns out that when there's duplicate headings, GitHub Markdown appends a `-1`, `-2`, etc to the heading names. So the Deployment `#requirements` ID becomes `#requirements-1`.